testRun.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+SCRIPT_PATH="./exokit.sh"
+
+# Execute ./exokit.sh -h and log the console output to testRunLog.txt,
+# stdout and stderr included
+"$SCRIPT_PATH" -h >testRunLog.txt 2>&1 &
+echo "Exokit loaded, Exit Code: " $?
+
+sleep 5s
+# Get process ID of exokit and kill it. (does not work )
+PID=$(pgrep -u root exokit.cmd)
+$(pkill -SIGKILL $PID)
+echo "Process killed, exit Code: " $?
+
+echo "Finished test run of ./exokit.sh -h"
+
+
+# keep console open for user to read or else goes away to quick
+read
```

Comment:
I probs wouldn't keep this open. Since this is for CI/testing I think it's ok to require this test to be run in a parent console.

Also if we hang like this then it will just hang CI and always fail.

Reply:
yes for sure I will take this out... more for dev use on my own machine.